OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -76,6 +76,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Vertical rate of turn.")]
 		public readonly int VerticalRateOfTurn = 6;
 
+		[Desc("Gravity applied while in free fall.")]
```

Comment:
Could you please add that the negative of this value is used internally?
Or better yet make the default `-10` and use whatever value the modder actually put in yaml.